shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -416,6 +423,9 @@ void EmbedderExternalViewEmbedder::SubmitFlutterView(
     GrDirectContext* context,
     const std::shared_ptr<impeller::AiksContext>& aiks_context,
     std::unique_ptr<SurfaceFrame> frame) {
+  // Get the view's cache, or create one if the view doesn't have a cache yet.
+  EmbedderRenderTargetCache& render_target_cache =
+      render_target_caches_[flutter_view_id];
```

Comment:
Maybe I'm missing something, but it doesn't look like this creates a cache if the view doesn't have one yet.

Reply:
`render_target_caches_` is a `unordered_map`. Its `[]` operator is defined so that, if it's used on an unrecognized index, the index is created with the default constructor.

Reply:
Oh I see. New comment looks good, thanks for clarifying!